packages/wrangler/src/queues/cli/commands/index.ts

```diff
@@ -22,6 +23,22 @@ export function queues(yargs: CommonYargsArgv) {
 		deleteHandler
 	);
 
+	yargs.command(
```

Comment:
is the `:` separator still an acceptable pattern?

Reply:
No we are actively trying to avoid that. In fact for v4 we are deprecating all of those and replacing them with non-colon commands.

Instead we prefer that we have a space between sub commands and their container. Also that the general structure should be `wrangler <noun> <verb>` - so in this case it feels to me like it would be more something along the lines of:

```
wrangler queues consumer http configure
wrangler queues consumer worker configure
```

Reply:
updated so `http` and `worker` are sub commands. So now we have:
```
wrangler queues consumer

Configure Queue Consumers

Commands:
  wrangler queues consumer add <queue-name> <script-name>     Add a Queue Worker Consumer
  wrangler queues consumer remove <queue-name> <script-name>  Remove a Queue Worker Consumer
  wrangler queues consumer http                               Configure Queue HTTP Pull Consumers
  wrangler queues consumer worker                             Configure Queue Worker Consumers
```
and: 
```
wrangler queues consumer http

Configure Queue HTTP Pull Consumers

Commands:
  wrangler queues consumer http add <queue-name>     Add a Queue HTTP Pull Consumer
  wrangler queues consumer http remove <queue-name>  Remove a Queue HTTP Pull Consumer
```

Reply:
cc @elithrar